src/screening/IdentifierFilters.jsx

```diff
@@ -77,8 +77,13 @@ class ScreeningFilters extends React.Component {
     this.handleSearch(null, e.target.value, 'second');
   }
 
+  getPrefixFromLocations = (locations) => {
+    const prefixes = R.map(R.path(['attributes', '0', 'value']))(locations);
```

Comment:
@jomadoye  did you see my comment on this line in the other pull request?

https://github.com/openmrs/openmrs-react-components/pull/111

Reply:
Yes @mogoodrich , I updated it.

Reply:
Sweet, thanks, I must have missed the follow-on commit...